templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -239,6 +239,13 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <files_dir path="/data/dnb11/galaxy_db/files"/>
             <extra_dir type="job_work" path="/data/jwd06/main"/>
         </backend>
+        <backend id="files32" type="disk" weight="2" store_by="uuid">
```

Comment:
```suggestion
        <backend id="files32" type="disk" weight="2" store_by="uuid" device="dnb11">
```

Reply:
I am assuming this `device=dnb11` is just a form of labeling.

Reply:
Yes, more like grouping, I have more of that in my other PR.